portal/integrated-windows-authentication/README.md

```diff
@@ -1,47 +1,41 @@
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), allows users to easily authenticate for different services using their domain's Windows credentials. This simplifies logging in to services that are provided through an intranet environment, as only a single username and password combination are required for the applications that support IWA.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
 
 ## How to use the sample
 
-Enter a URL to your IWA-secured portal into the URL field and select "Search IWA Secured Portal" to search for web maps stored there. Authenticate with a username (including domain, such as username@DOMAIN or domain\username), and password. If authentication is successful, portal item results will display in the list. Select a web map item to display it in the map view.
+Enter the URL to your IWA-secured portal. Tap the button to search for web maps stored on the portal. You will be prompted for a user name, password, and domain. If you authenticate successfully, portal item results will display in the list. Select a web map item to display it in the map view.
 
 ## How it works
 
-1. Set up an `AuthenticationChallengeHandler` that challenges the user for authentication, such as the custom handler created for this sample: `IWAChallengeHandler`.
-2. Set the `AuthenticationManager` to use the created challenge handler using `.setAuthenticationChallengeHandler(new IWAChallengeHandler());`.
-3. Create a new `Portal` from a portal URL and a boolean indicating if it is a secured resource using `new Portal(url, true)`.
-4. Load the portal. The authentication challenge handler will prompt the user for credentials. The portal will load successfully with valid credentials.
-5. Create `PortalQueryParameters` to define a search query for map items in the portal.
-5. Find web map portal items using `Portal.findItemsAsync(portalQueryParameters)`.
-6. Create a map using a `PortalItem` result with `new ArcGISMap(portalItem)`.
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
+2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `UserCredential` object from the information entered by the user.
+3. If the user authenticates, the search returns a list of web map `PortalItem`s and the user can select one to display as an `ArcGISMap`.
 
 ## Relevant API
 
-* AuthenticationChallenge
-* AuthenticationChallengeHandler
-* AuthenticationChallengeResponse
```

Comment:
I know these aren't mentioned in the 'How it works' section any more, but I think they are quite fundamental to this sample. Considering this section would power the search and would help users find these samples, perhaps we should leave them in?